build_windows.sh

```diff
@@ -47,7 +47,7 @@ make VERBOSE=1
 # Finally, build freedv-gui
 cd $FREEDVGUIDIR && git pull
 mkdir -p $BUILD_DIR && cd $BUILD_DIR && rm -Rf *
-$CMAKE -DCMAKE_BUILD_TYPE=Debug -DCODEC2_BUILD_DIR=$CODEC2DIR/$BUILD_DIR -DLPCNET_BUILD_DIR=$LPCNETDIR/$BUILD_DIR -DBOOTSTRAP_WXWIDGETS=1 ..
+$CMAKE -DCMAKE_BUILD_TYPE=Debug -DCODEC2_BUILD_DIR=$CODEC2DIR/$BUILD_DIR -DLPCNET_BUILD_DIR=$LPCNETDIR/$BUILD_DIR ..
 make VERBOSE=1
```

Comment:
@hobbes1069 - this caused the Windows build to choke.  If you are keen, pls feel free to fix, however I'm content to build against the default Fedora wxWidgets for now.

Reply:
Definitely something broken. I just got back from a Boy Scout canoe trip. May not be able to fix tonight but I will investigate.